packages/opentelemetry-node/package.json

```diff
@@ -42,12 +42,14 @@
   },
   "devDependencies": {
     "@opentelemetry/context-base": "^0.8.0",
+    "@opentelemetry/plugin-grpc": "^0.8.0",
```

Comment:
You don't need to load the actual grpc plugin for the config. You just need to test that your merge config function doesn't overwrite options for other plugins.

```typescript
const defaultConfig = {
  module1: {
    enabled: true,
    path: 'testpath'
  },
  module2: {
    enabled: true,
    path: 'testpath2'
  }
};

const userConfig = {
  module2: {
    path: 'userpath'
  }
}

const merged = mergeConfig(defaultConfig, userConfig);

// assert merged.module1.path is testpath
// assert merged.module2.path is userpath
```

Reply:
Thanks @dyladan, I did not think about factoring out `mergePlugins` (which I call it as such since the types it merges and returns are `Plugins` rather than plugin configs.)

I've exported it for the sole purpose of importing it in the test suite.